confluent/docker_utils/cub.py

```diff
@@ -261,6 +263,68 @@ def check_connect_ready(host, port, service_timeout):
         return False
 
 
+def check_ksql_server_ready(host, port, service_timeout):
+    """Waits for KSQL server to be ready.
+
+    Args:
+        host: Hostname where KSQL server is hosted.
+        port: KSQL server port.
+        timeout: Time in secs to wait for the service to be available.
+
+    Returns:
+        False, if the timeout expires and KSQL server is not ready, True otherwise.
+
+    """
+
+    # Check if you can connect to the endpoint
+    status = wait_for_service(host, port, service_timeout)
+
+    if status:
+        # Check if service is responding as expected to basic request
+        url = "http://%s:%s" % (host, port)
+        r = requests.get(url)
+        # The call should always return a json string including version
+        if r.status_code // 100 == 2 and 'version' in str(r.text):
```

Comment:
The return string looks like this: 

```
{
    "KsqlServerInfo": {
        "kafkaClusterId": "A2xvXUXdTUKBhaQjdgYZPg",
        "ksqlServiceId": "confluent_rmoff_01",
        "version": "5.2.1"
    }
}
```

I'd suggest matching on something more specific than `version`, e.g. `KsqlServerInfo`.

Reply:
I intentionally did not because of a concern (admittedly not founded in data) that the string `KsqlServerInfo` might be more likely to change than `version`...even if the chance of this happening is very small.  The bet is the response will always contain `version`

Reply:
I think we should match on `Ksql`. Otherwise it could be another service on that port that happens to return `version`.

Reply:
Done